opm/simulators/flow/python/PyBlackOilSimulator.hpp

```diff
@@ -0,0 +1,76 @@
+/*
+  Copyright 2020 Equinor ASA.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef OPM_PY_BLACKOIL_SIMULATOR_HEADER_INCLUDED
+#define OPM_PY_BLACKOIL_SIMULATOR_HEADER_INCLUDED
+
+#include <opm/simulators/flow/Main.hpp>
+#include <opm/simulators/flow/FlowMainEbos.hpp>
+#include <opm/models/utils/propertysystem.hh>
+#include <opm/simulators/flow/python/Pybind11Exporter.hpp>
+#include <opm/simulators/flow/python/PyMaterialState.hpp>
+#include <opm/parser/eclipse/Deck/Deck.hpp>
+#include <opm/parser/eclipse/EclipseState/EclipseState.hpp>
+#include <opm/parser/eclipse/EclipseState/Schedule/Schedule.hpp>
+#include <opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.hpp>
+
+namespace Opm::Pybind {
+class PyBlackOilSimulator
+{
+    using TypeTag = Opm::Properties::TTag::EclFlowProblem;
+private:
```

Comment:
The `TypeTag` is implicitly `private:` too.  Maybe you'd like to explicitly declare it as such?